npm/pkg/controlplane/controllers/v2/networkPolicyController.go

```diff
@@ -230,7 +230,7 @@ func (c *NetworkPolicyController) syncNetPol(key string) error {
 		// netPolController does not need to reconcile this update.
 		// In this updateNetworkPolicy event,
 		// newNetPol was updated with states which netPolController does not need to reconcile.
-		if reflect.DeepEqual(*cachedNetPolSpecObj, netPolObj.Spec) {
+		if reflect.DeepEqual(cachedNetPolSpecObj, &netPolObj.Spec) {
```

Comment:
Curious - Are their result different?
`reflect.DeepEqual(*cachedNetPolSpecObj, netPolObj.Spec)` and `reflect.DeepEqual(cachedNetPolSpecObj, &netPolObj.Spec)`

Reply:
Yeah they were different, deepEqual returned false early since the types did not match